src/main/twirl/uk/gov/hmrc/play/views/layouts/EuExitLinks.scala.html

```diff
@@ -21,8 +21,9 @@
 <h2 class="heading-medium">@Messages("eu-exit.links.heading.text")</h2>
 <hr>
 <div class="grid-row">
-  <ul class="push--bottom">
-    <div class="column-one-half">
+
+  <div class="column-one-half">
```

Comment:
Do you need the div? Can you just add the class to the `<ul>` without changing the styling?